unison-src/transcripts/api-getDefinition.md

```diff
@@ -1,43 +1,44 @@
 # Get Definitions Test
 
 ```ucm:hide
-.> builtins.mergeio
+.nested> builtins.mergeio
 ```
 
 ```unison:hide
 {{ Documentation }}
-nested.names.x = 42
+names.x = 42
 ```
 
 ```ucm:hide
-.> add
+.nested> add
 ```
 
 ```api
 -- Should NOT find names by suffix
 GET /api/non-project-code/getDefinition?names=x
 
 -- Term names should strip relativeTo prefix.
-GET /api/non-project-code/getDefinition?names=x&relativeTo=nested
+GET /api/non-project-code/getDefinition?names=names.x&relativeTo=nested
 
 -- Should find definitions by hash, names should be relative
 GET /api/non-project-code/getDefinition?names=%23qkhkl0n238&relativeTo=nested
+```
 
--- Should filter out any definitions which aren't in the provided namespace even if the hash matches.
```

Comment:
This was a bit weird, I don't think the behaviour on trunk is consistent with this, but also don't think we really care since the frontend shouldn't be asking for things that are out of scope (and if it does, I guess it's fine if it renders with hashes 🤷🏼 ). I'm just going to delete this particular test since I don't think it makes sense.

This doesn't affect Share since that uses a completely different api implementation.